Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/CheckVersionIntegrityCommand.cs

```diff
@@ -59,16 +62,20 @@ public override void Execute(CancellationToken cancellationToken)
 
             DebugLogger.Log("Checking version integrity.");
 
+            _status.IsActive.Value = true;
+
             var files = new FileIntegrity[_versionSummary.Files.Length];
 
             for (int i = 0; i < _versionSummary.Files.Length; i++)
             {
                 files[i] = CheckFile(_versionSummary.Files[i]);
 
-                _statusReporter.OnProgressChanged((i + 1)/(double)_versionSummary.Files.Length, "Checking version integrity...");
+                _status.Progress.Value = (i + 1) / (double) _versionSummary.Files.Length;
```

Comment:
Possible division by 0. Do assert-like check somewhere.

Reply:
It's not possible - if the `_versionSummary.Files.Length` equals to `0` then this loop is not even executed.